packages/lektor_atom_plugin/setup.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+"""
+This is an updated copy of lektor-atom plugin
+wich replace deprecated Werkzeug's atom with Pelican's feedgenerator .
```

Comment:
Minor typos: "which replaces the deprecated Werkzeug's atom with Pelican's feedgenerator."